relayer/crates/starknet-test-components/src/impls/bootstrap/start_chain.rs

```diff
@@ -48,6 +48,7 @@ where
             "full",
             "--dump-on",
             "block",
+            "--request-body-size-limit=3000000", // 3M; ibc-core contract class is too large
```

Comment:
Wouldn't this also cause issues in mainnet later on?

Reply:
I could deploy ibc-core class with 2.2M size on testnet. 
https://sepolia.starkscan.co/tx/0x031650ea44dce487cba07282a5dfae7ae3f9db229394b7577ccff4a79e90dc20

I think, it's just a node configuration.